packages/frontend/src/renderer/components/LoadingPanel/LoadingPanel.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from 'react'
+import { useSelector } from 'react-redux'
+import { useModal } from '../../containers/hooks'
+import { ModalName } from '../../sagas/modals/modals.types'
+import { socketSelectors } from '../../sagas/socket/socket.selectors'
+import { communities, publicChannels } from '@quiet/nectar'
+import LoadingPanelComponent from './loadingPanelComponent'
+
+export enum LoadingPanelMessage {
+  Loading = 'Loading...',
+  StartingApplication = 'Starting Quiet',
+  FetchingData = 'Fetching Data'
+}
+
+const LoadingPanel = () => {
+  const loadingPanelModal = useModal(ModalName.loadingPanel)
+
+  const isConnected = useSelector(socketSelectors.isConnected)
```

Comment:
We shouldn't use selectors in components. There should be corresponding container.

Reply:
It is container